beacon-chain/types/block.go

```diff
@@ -1,72 +1,120 @@
 package types
 
 import (
+	"fmt"
 	"hash"
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+	pb "github.com/prysmaticlabs/prysm/proto/sharding/v1"
 	"golang.org/x/crypto/blake2b"
 )
 
 // Block defines a beacon chain core primitive.
 type Block struct {
-	data *Data
+	data                  *pb.BeaconBlockResponse
+	activeStateHash       hash.Hash
```

Comment:
why aren't `activeStateHash` and `crystallizedStateHash` part of `data`?

Reply:
I think they are, `activeStateHash ` and `crystallizedStateHash` does not need to be added to the block struct over here, we can get them from `data`